tests/specs/analog/uplinkCommands.spec.js

```diff
@@ -0,0 +1,111 @@
+import {test, expect} from '@playwright/test';
+import {MainPage} from '../../objects/MainPage.js';
+import fixture from '../../fixtures/main.js';
+import {uplinkComplexCommands, uplinkSimpleCommands} from '../../fixtures/analog/uplinkCommands.js';
+
+
+const validateUplinkCommands = async ( page, format, command, isComplex = false ) => {
```

Comment:
Why command here it's just a name, not a real command data? With dump, parameters, etc.

Reply:
also function with name "validateUplinkCommands" and argument "command" looks inappropriate

Reply:
fixed